inference-engine/tests/ie_test_utils/common_test_utils/test_common.cpp

```diff
@@ -67,6 +68,13 @@ TestsCommon::TestsCommon() {
         std::cout << "\nMEM_USAGE=" << memsize << "KB\n";
     }
     InferenceEngine::ExecutorManager::getInstance()->clear();
+
+    auto seed_str = std::getenv("OPENVINO_TEST_SEED");
```

Comment:
But tests should reproduce the same behavior in CI and local. It can become the main reason of a lot of sporadic issues. 
Possible decision is using of seed of a test argument (and run it with non-default values in nightly). But it is just first idea came to mind